mailpoet/mailpoet
# Offer to download or activate the premium plugin - MAILPOET-3971

Offer to download or activate the premium plugin or upgrade plan in premium banner

[MAILPOET-3971](https://mailpoet.atlassian.net/browse/MAILPOET-3971)

Related to https://github.com/mailpoet/shop/pull/669

Testing instructions:
1. Make sure to downgrade Premium plugin to be 3.77.0 in order to test this pull request as this PR is on 3.77.0. Please head over [to this page to download](https://github.com/mailpoet/mailpoet-premium/releases) and upload 3.77.0 build. After that follow the next steps.
2. Go to MailPoet > Emails page
3. Click on statistics link for any of the sent newsletters to access the statistics page
4. On this page you could see the banner that is mentioned in the Jira ticket above. Please follow all three cases from the [Jira specification](https://mailpoet.atlassian.net/browse/MAILPOET-3971) to verify that the banner is present with the appropriate content is in the banner.
5. Three cases to test: With inactive Premium plugin; With not installed Premium plugin; With applied lower-tier subscription (go to call for testing p2 post to find the key to be applied before testing (under IMPORTANT NOTES BEFORE TESTING & SUBSCRIPTION KEYS section));
